### DISCUSSION TITLE
"Relax" mod should have decreased score multiplier

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#19164] [Q&A]
This discussion was created simply to point out the problem that the current multiplier system will be unfair in most cases. In my case, I "turned on" the mods, but set their settings to the most convenient for me, but got the same multiplier. We need to somehow come up with systems to more accurately determine the multiplier relative to active mods and their exact settings.

Yes, it's fun. But this should be corrected in the near future. I don't think relax should always give a 1x multiplier. This discussion is more of a "call to arms".
![image](https://user-images.githubusercontent.com/59089206/179367963-0252baef-bbd9-4353-9d44-0ec3d5a153e0.png)
![image](https://user-images.githubusercontent.com/59089206/179367974-507f7c48-a723-4a78-93c5-aa5d1a9e0fe6.png)

**Comment by NotGumballer91** [comment]
I personally say any scores with Automation mods should not be on the leaderboards.

**Comment by Theighlin** [comment]
Duplicate of #7698, though i think there was a more recent discussion with more insight on the matter (or a community meeting).

**Reply by Moltanicaa** [reply]
You are right, thanks. It's time to freshen it up.

**Reply by apollo-dw** [reply]
The scope of this discussion is a little bigger than the linked issue, since that one only is referring to the score multiplier on rate mods.
